tower-http/src/redirect.rs

```diff
@@ -0,0 +1,100 @@
+//! Service that redirects all requests.
```

Comment:
nit: an example might be nice...but we can do that later.

Reply:
👍